### PR TITLE
Make busty retry on "busy rn"

### PR DESCRIPTION
**Pull request by Cephian**
Make busty try one more time when she gets a timeout from openai before resorting to "busy rn"

### CODE REVIEW

File: src/llm.py [hunk 1]
```diff
@@ -291,15 +292,15 @@ async def get_response_text(message: Message) -> Optional[str]:
...
 
-    response = await query_api(data)
-    if response:
-        # Remove "Busty: " prefix
-        prefix = f"{get_name(self_user)}: "
-        if response.startswith(prefix):
-            response = response[len(prefix) :]
-
-        return response
+    for x in range(config.LLM_RETRY):
```
Comment (anoadragon453) [inline src/llm.py:295]:
```suggestion
    for _ in range(config.LLM_RETRY):
```

If we're not using the variable, it's more common to use `_`.

File: src/config.py [hunk 2]
```diff
@@ -31,6 +31,8 @@
 PREVIEW_JUMP_URL = "https://youtu.be/J45GvH2_Ato"
 # The OpenAI model to use for GPT abilities
 OPENAI_MODEL = "gpt-3.5-turbo"
+# Number of times to retry if API does not respond in time
+LLM_RETRY = 2
```
Comment (anoadragon453) [inline src/config.py:35]:
```suggestion
LLM_RETRY_COUNT = 2
```

may be slightly clearer.

**Review by anoadragon453 — APPROVED** [review]
tiny tweaks. lgtm otherwise!

**Comment by Cephian** [comment]
OpenAI seems to have gotten their act together and this hardly happens anymore, so I'm just gonna close this as it would probably add needless bloat to the already chaotic LLM module. If it starts happening again we can reconsider.

**Pull request closed without merging**